app/src/main/java/com/codesoom/assignment/application/UserServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.NoSuchElementException;
+
+/**
+ * 사용자 생성, 수정, 삭제 기능을 담당하는 클래스
+ */
+@Transactional
+@Service
+public class UserServiceImpl implements UserService {
+
+    private final UserRepository userRepository;
+
+    public UserServiceImpl(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * 사용자를 생성하여 데이터베이스에 저장하는 메소드
+     * @param source 사용자 객체에 매핑하기 위한 객체
+     * @return 사용자 객체
+     */
```

Comment:
```suggestion
    /**
     * 사용자를 생성하고, 생성된 사용자 정보를 리턴합니다.
     * @param source 생성할 사용자 정보
     * @return 생성된 사용자
     */
```

- 이 클래스는 `repository`로 캡슐화된 영속성 계층과 관계를 맺고 있어요. 따라서 데이터를 저장하는 곳이 `데이터베이스`인지 `파일`인지 `테이프 레코더`인지는 숨겨져 있습니다. 그래서 `데이터베이스`를 주석에서 의식하지 않아도 됩니다.
- 메소드에 대한 설명이므로 `메소드`라고 굳이 설명하지 않아도 괜찮아요.

Reply:
javadoc 과 테스트 코드의 DisplayName 의 설명을 작성하는게 생각보다 어려운 것 같습니다. 이해하기 쉽도록 적으려고 하는데 항상 문장이 자연스럽지 못한것 같아요 ㅠ 더 연습해 보겠습니다